docs/src/api/class-request.md

```diff
@@ -77,6 +77,17 @@ Headers with multiple entries, such as `Set-Cookie`, appear in the array multipl
 An array with all the request HTTP headers associated with this request. Unlike [`method: Request.allHeaders`], header names are not lower-cased.
 Headers with multiple entries, such as `Set-Cookie`, appear in the array multiple times.
 
+## async method: Request.headerValue
```

Comment:
Should it also have `Request.headerValues` ?

Reply:
no, because there are never multiple headers with the same name on a request.